src/services/resource-service.ts

```diff
@@ -47,6 +58,16 @@ export const ResourceService = {
     params?: GetResourcesParams
   ): Promise<AxiosResponse<ItemsWithCount<Quiz>>> =>
     await axiosClient.get(URLs.quizzes.get, { params }),
+
+  getQuizzesQuery: (params?: GetResourcesParams) => {
+    return baseService.request<ItemsWithCount<Quiz>>({
+      method: 'GET',
+      url: getFullUrl({
+        pathname: URLs.quizzes.get,
+        searchParameters: params
```

Comment:
Are you sure that it's working without destructuring?
```suggestion
        searchParameters: { ...params }
```